iaso/api/mobile/org_unit_change_requests.py

```diff
@@ -0,0 +1,54 @@
+from rest_framework import viewsets, serializers
+from rest_framework.mixins import ListModelMixin
+
+from iaso.api.org_unit_change_requests import HasOrgUnitsChangeRequestPermission
+from iaso.api.serializers import AppIdSerializer
+from iaso.api_filters.org_unit_change_requests import MobileOrgUnitChangeRequestListFilter
+from iaso.models import OrgUnitChangeRequest, OrgUnit
+
+
+class MobileOrgUnitChangeRequestListSerializer(serializers.ModelSerializer):
+    org_unit_id = serializers.IntegerField(source="org_unit.id")
+    org_unit_uuid = serializers.UUIDField(source="org_unit.uuid")
+
+    class Meta:
+        model = OrgUnitChangeRequest
+        fields = [
+            "id",
+            "org_unit_id",
+            "org_unit_uuid",
+            "status",
+            "approved_fields",
+            "rejection_comment",
+            "created_at",
+            "updated_at",
+            "new_parent_id",
+            "new_name",
+            "new_org_unit_type_id",
+            "new_groups",
+            "new_location",
+            "new_accuracy",
+            "new_reference_instances",
+        ]
+
+
```

Comment:
Is it the use of listmodelmixin that enforces that only GET is available ?

Reply:
Yes, it's documented in "[Custom ViewSet base classes > Example](https://www.django-rest-framework.org/api-guide/viewsets/#example_3)".